src/catalog/src/system_schema/information_schema/key_column_usage.rs

```diff
@@ -236,6 +240,30 @@ impl InformationSchemaKeyColumnUsageBuilder {
                             1, //always 1 for time index
                         );
                     }
+                    if column.is_inverted_indexed() {
```

Comment:
A column can have an inverted index, be part of the primary key, and also have a full-text index. I believe it's preferable to combine the key information rather than add a new row.

Reply:
this is a better approach what would be the best aggregation approach?
I am thinking consolidate it separated by comma such as indextype
`greptime-fulltext-index-v1, greptime-inverted-index-v1`
or CONSTRAINT_NAME column as
`PRIMARY, INVERTED INDEX, FULLTEXT INDEX`

one more assumption - the time index cannot coexist with fulltext index or inverted index right? I dont have to worry about the conflicts in ordinal position and the Computed "is index nullable" column.

Reply:
> I am thinking consolidate it separated by comma such as indextype
> `greptime-fulltext-index-v1, greptime-inverted-index-v1`
> or CONSTRAINT_NAME column as
> `PRIMARY, INVERTED INDEX, FULLTEXT INDEX`

Yes, it's my preferred way, WDYT @waynexia

Reply:
updated